ZUGFeRD/InvoiceDescriptor23CIIWriter.cs

```diff
@@ -528,7 +528,7 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFo
                             #region ChargePercentage
                             if (specifiedTradeAllowanceCharge.ChargePercentage.HasValue)
                             {
-                                Writer.WriteStartElement("ram", "CalculationPercent", ALL_PROFILES ^ Profile.Basic); // BT-138, BT-143
+                                Writer.WriteStartElement("ram", "CalculationPercent"); // BT-138, BT-143

```

Comment:
can you please exclude Minimum and Basic WL according to:

![grafik](https://github.com/user-attachments/assets/13102315-32b7-4247-9c02-b72051cf8236)